pom.xml

```diff
@@ -19,31 +19,59 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>io.openshift.booster</groupId>
-    <artifactId>spring-boot-booster-parent</artifactId>
-    <version>1.5.13-2</version>
+    <groupId>io.openshift</groupId>
```

Comment:
I thought the idea was to get rid of all the parents?

Reply:
That's the final goal. @metacosm and I agreed to take it one step at a time